### DISCUSSION TITLE
possible to Download directly to remote server?

### DISCUSSION THREAD

**Discussion by csss1234** [mxpv/podsync#310] [Ideas]
Right now I’m running podsync cli on Mac and adding the downloads to a public pcloud.com folder so I can listen anywhere. 

I have a slow internet connection, so i try to limit all Downloading and uploading in general.

I pay for shared hosting of some websites, with unlimited storage space. Because it’s shared, I can’t install docker or ffmpeg etc on the server. 

Is there a way to config podsync to request the server download the files directly? (I am able to do this manually with ssh and wget.)

**Comment by Rpsl** [comment]
In general you can mount your remote folder as native folder to your mac, but anyway files must downloaded to your mac and after that upload to server.

If you want to download files directly to server you need run podsync on the server.

**Comment by mxpv** [comment]
Remote storages is on the list.
I plan to implement S3 storage soon.
See https://github.com/mxpv/podsync/issues/180
